browser/resources/settings/default_brave_shields_page/default_brave_shields_page.ts

```diff
@@ -103,6 +103,13 @@ class BraveShieldsPage extends BraveShieldsPageBase {
       }
     })
 
+  }
+
+  refreshState() {
+    this.browserProxy_.getHTTPSEverywhereEnabled().then(value => {
+      this.httpsEverywhereEnabled_ = value
+    })
+    
```

Comment:
I might not be understanding, but something doesn't seem quite right here to me. Where is `refreshState` called? I also don't see a definition for `getHTTPSEverywhereEnabled`.

Reply:
If I'm right, it's most likely the root cause for these two issues:
- https://github.com/brave/brave-browser/issues/26068
- https://github.com/brave/brave-browser/issues/26067